pkg/bootstrap/types.go

```diff
@@ -27,13 +27,19 @@ type KuryrBootstrapResult struct {
 	UserCACert               string
 }
 
+type OVNConfigBoostrapResult struct {
```

Comment:
a quick docblock would be nice. Also, naming this OVNConfigState might be better (but that's bike-shedding).